### ISSUE TITLE
App-Bar search component text does not have enough contrast

### ISSUE THREAD

**Issue by MichaelHurd** [tyler-technologies-oss/forge#796]
**Describe the bug:**
The placeholder text in the app-bar search component fails ADA accessibility guidelines for contrast.

**To Reproduce:**
Take a screenshot of the app-bar's search component.
Check the color values using an eyedropper tool.
The values have a contrast ratio of only ~2.57:1. WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1

**Expected behavior:**
The text should have more contrast.

**Screenshots:**
![Image](https://github.com/user-attachments/assets/313b5fb4-5276-4910-be36-f6b43417cebb)

**Please complete the following information:**
 - Forge version: v3.4.2
 - I have searched existing issues before creating this report? Y
 - Browser: All
 - Platform: All
 - OS: All
